modules/tide_edit_protection/tide_edit_protection.install

```diff
@@ -46,3 +47,95 @@ function tide_edit_protection_update_8001() {
   $contentTimeOutSettings->set('content_lock_timeout_on_edit', 1);
   $contentTimeOutSettings->save();
 }
+
+/**
+ * Adds node site filter to lock_content view.
+ */
+function tide_edit_protection_update_8002() {
+  if (!\Drupal::moduleHandler()->moduleExists('tide_site_restriction')) {
+    return;
+  }
+  $site_filter =
+    [
+      'id' => 'field_node_site_target_id',
+      'table' => 'node__field_node_site',
+      'field' => 'field_node_site_target_id',
+      'relationship' => 'none',
+      'group_type' => 'group',
+      'admin_label' => '',
+      'operator' => 'or',
+      'value' => [],
+      'group' => 1,
+      'exposed' => TRUE,
+      'expose' =>
+        [
+          'operator_id' => 'field_node_site_target_id_op',
+          'label' => 'Sites',
+          'description' => '',
+          'use_operator' => FALSE,
+          'operator' => 'field_node_site_target_id_op',
+          'operator_limit_selection' => FALSE,
+          'operator_list' => [],
+          'identifier' => 'field_node_site_target_id',
+          'required' => FALSE,
+          'remember' => FALSE,
+          'multiple' => TRUE,
+          'remember_roles' =>
+            [
+              'authenticated' => 'authenticated',
+              'anonymous' => '0',
+              'administrator' => '0',
+              'approver' => '0',
+              'site_admin' => '0',
+              'editor' => '0',
+              'previewer' => '0',
+              'event_author' => '0',
+              'grant_author' => '0',
+            ],
+          'reduce' => FALSE,
+        ],
+      'is_grouped' => FALSE,
+      'group_info' =>
+        [
+          'label' => '',
+          'description' => '',
+          'identifier' => '',
+          'optional' => TRUE,
+          'widget' => 'select',
+          'multiple' => FALSE,
+          'remember' => FALSE,
+          'default_group' => 'All',
+          'default_group_multiple' => [],
+          'group_items' => [],
+        ],
+      'reduce_duplicates' => FALSE,
+      'type' => 'select',
+      'limit' => TRUE,
+      'vid' => 'sites',
+      'hierarchy' => TRUE,
+      'error_message' => TRUE,
+      'plugin_id' => 'taxonomy_index_tid',
+    ];
+
+  $config = \Drupal::configFactory()
+    ->getEditable('views.view.tide_locked_content')
+    ->get('display');
+  if (!isset($config['default']['display_options']['filters']['field_node_site_target_id'])) {
```

Comment:
If you are verifying 'field_node_site_target_id' exists why don't you verify default, display_options and filters as well? Just I am wondering that.

Reply:
yeah, in this case, the logic behind,  the only thing that I should care about is just whether  `field_node_site_target_id` exist or not, as I assume that the clients who enabled the module should have the view.  if it doesn't, it is not our responsibility.